<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Class18Class</class>
 <widget class="QDialog" name="Class18Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Class18</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton1">
       <property name="text">
        <string>Cats</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton2">
       <property name="text">
        <string>Dogs</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Click me</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="class18.qrc"/>
 </resources>
 <connections/>
</ui>
